src/components/Navbar/WelcomeBoard.tsx

```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import '../RoomView/card.css';
```

Comment:
originally a mistyped import. 
this file doesn't exist anymore so i had to delete the import.